README.md

```diff
@@ -384,6 +384,7 @@ To add a resource or contribute, please check out our contribution guide [here](
 - [Introduction to Linux](https://training.linuxfoundation.org/resources/free-courses/introduction-to-linux/)\
 - [Linux kernel and its insides](https://0xax.gitbooks.io/linux-insides/content/)
 - [Interactive map of Linux Kernel](https://makelinux.github.io/kernel/map/)
+- [Linux Device Driver Development](https://lwn.net/Kernel/LDD3/)
```

Comment:
Please replace the title with the page title

Reply:
I have changed the title with page title.